go.mod

```diff
@@ -15,4 +15,48 @@ require (
 	go.uber.org/atomic v1.6.0
 )
 
+require (
+	github.com/blang/semver v3.5.1+incompatible // indirect
+	github.com/davecgh/go-spew v1.1.1 // indirect
+	github.com/dyatlov/go-opengraph v0.0.0-20180429202543-816b6608b3c8 // indirect
```

Comment:
why all these packages are added?

Reply:
All the packages are indirect dependency of the direct dependencies. 
All these got added after the go version was changed from 1.16 to 1.18

Reply:
have we run the command `go mod tidy`?

Reply:
yes